ape_safe/accounts.py

```diff
@@ -312,6 +364,20 @@ def call(  # type: ignore[override]
 
         return super().call(txn, **call_kwargs)
 
+    def get_api_confirmations(self, safe_tx: SafeTx) -> Dict[AddressType, MessageSignature]:
+        safe_tx_hash = hash_eip712_message(safe_tx).hex()
```

Comment:
To avoid a bug with `deploy` transactions here, I had to do this:

```python
safe_tx.to = safe_tx.to or ZERO_ADDRESS
safe_tx_hash = hash_eip712_message(safe_tx).hex()
```